seqr/views/utils/variant_utils.py

```diff
@@ -60,6 +61,21 @@ def get_variant_key(xpos=None, ref=None, alt=None, genomeVersion=None, **kwargs)
     return '{}-{}-{}_{}'.format(xpos, ref, alt, genomeVersion)
 
 
+def saved_variant_genes(variants):
```

Comment:
This is copy-pasted from the api file into the utility file since it is shared in multiple places. Original here: https://github.com/macarthur-lab/seqr/pull/1352/files#diff-f34dfe2f07218b25094615c1bb2c39cbL350